### ISSUE TITLE
storyboard game state triggers not useful

### ISSUE THREAD

**Issue by Karoo13** [ppy/osu-stable-issues#395]
According to the [game state definition](https://osu.ppy.sh/help/wiki/Storyboard_Scripting/General_Rules#game-state) in standard:

1. During gameplay it is based on whether the previous combo was SSed.
2. During and after breaks, it is based on whether hp is above or below half.
3. After the map ends, it is based on whether half of the breaks were in the pass state, or if there were no breaks, see point 2.

I don't think the first part about SSing combos is useful to many storyboarders, while adjusting the storyboard based on low hp during gameplay would be useful. Since the goal is to judge whether the player is doing poorly, low hp is a better indicator of this than getting a single 100. If the storyboarder wants to judge low acc on a specific object, it can be done with hitsounds, while a trigger based on transition to fail state requires that they had been SSing before that combo, so it is unreliable.

Basically, remove the first criterion, and maybe the third as well.

**Comment by peppy** [comment]
This is something which came from Osu! Tatakae! Ouendan!. We can consider changing the conditions in lazer, but I'm not willing to make a change to stable. If you want to see it changed in lazer, please open an issue against the lazer project (although please first wait for triggers to be implemented, as i don't think they are yet).
